hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/webapp/RouterWebServices.java

```diff
@@ -106,7 +106,7 @@
  * main difference with AMRMProxyService is the protocol they implement.
  **/
 @Singleton
-@Path("/ws/v1/cluster")
```

Comment:
`/ws/v1/cluster` should be replaced with the variable `RMWSConsts.RM_WEB_SERVICE_PATH`.